drift/openapi/admin_api.spec.yaml

```diff
@@ -0,0 +1,36 @@
+---
+openapi: 3.0.1
+info:
+  version: "0.1"
+  title: Drift Backend Service Admin Plane
+  description: Admin API for Drift Service
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+servers:
+  - url: "{{ path_prefix }}{{ app_name }}/admin"
+
+
+paths:
```

Comment:
I feel like I recall the version being required, I don't know if there would be an exception for admin, but here is an example:
https://github.com/RedHatInsights/system-baseline-backend/blob/master/system_baseline/openapi/internal_api.spec.yaml#L14-L26

Reply:
During the design process we decide to move the admin without any version, because admin will be used only by managers etc.. and will not have several changes

Reply:
Ok, in that case, if it works locally and passes automation and schema checks, then that's fine.